composer.json

```diff
@@ -2,55 +2,21 @@
     "name": "maleficarum/api",
     "description": "API component",
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "ext-mbstring": "*",
         "ext-PDO": "*",
```

Comment:
Please remove PDO from required extensions. It should not be required by API component